ci/axis/dask.yaml

```diff
@@ -21,27 +16,4 @@ RAPIDS_VER:
   - '21.08'
   - '21.10'
 
-UCX_PY_VER:
-  - '0.21'
-  - '0.22'
-
-excludes:
-  - DOCKER_FILE: dask.Dockerfile
-    BUILD_IMAGE: gpuci/distributed
-  - DOCKER_FILE: dask.Dockerfile
-    BUILD_IMAGE: gpuci/dask_sql
-
-  - DOCKER_FILE: distributed.Dockerfile
-    BUILD_IMAGE: gpuci/dask
-  - DOCKER_FILE: distributed.Dockerfile
-    BUILD_IMAGE: gpuci/dask_sql
-
-  - DOCKER_FILE: dask_sql.Dockerfile
-    BUILD_IMAGE: gpuci/dask
-  - DOCKER_FILE: dask_sql.Dockerfile
-    BUILD_IMAGE: gpuci/distributed
-
-  - RAPIDS_VER: '21.08'
-    UCX_PY_VER: '0.22'
-  - RAPIDS_VER: '21.10'
-    UCX_PY_VER: '0.21'
\ No newline at end of file
+excludes:
```

Comment:
Is it necessary to have an empty `excludes` section or could it be removed?

Reply:
Not sure - my assumption was that it should be included, as we also include it in the [axis file](https://github.com/dask/dask/blob/main/continuous_integration/gpuci/axis.yaml) for Dask's gpuCI, but ops should be able to answer this question.

Reply:
@raydouglass do you know if an empty `excludes` section is mandatory or if we could remove it?

Reply:
It is mandatory, it just be left empty though